doc/operator-upgrade.md

```diff
@@ -115,6 +115,18 @@ The upgrade to Operator 0.5.0 can be triggered using the following command
 helm upgrade <operator release name> <location of 0.5.0 charts> --set webhookCert.crt=<tls.crt> --set webhookCert.generate=false --set webhookCert.certName=<cert-name> --set webhookCert.secretName=<secret-name>
 ```
 
+Once the upgrade completes and the following command starts returning the expected response
```

Comment:
Could be useful to know what "the expected response" was (i.e. what to look for exactly)

Reply:
done